app/models/utils/load_testing/sms_submission_load_test.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+module Utils
+  module LoadTesting
+    # Builds a JMeter test plan for submitting a lot of SMS form submissions.
+    #
+    # This test building process should be run on the server that is being targeted
+    # for the test because it will query the database while building the sample submission
+    # data (i.e. the message body for the SMS submission) for the test.
+    #
+    # The actual JMeter test will be run on a different server that does not have NEMO installed.
+    # It will just know how to make a bunch of HTTP requests based on the options and data we give it.
+    class SmsSubmissionLoadTest < LoadTest
+      # Required `options`:
+      #   user_id: User doing the submitting
+      #   form_id: The ID of the form being submitted to.
+      #
+      # The test currently knows how to handle the following question types:
+      #   text, long_text, integer, counter, decimal, select_one,
+      #   select_multiple, datetime, date, time
+      #
+      # * The submitting user must have a phone number and SMS auth code
+      # * The Twilio adapter must be configured appropriately on the target
+
+      CSV_FILENAME = "submissions.csv"
+
+      # Generates a bunch of test data to be used to make the test requests and stores it in a CSV file.
+      def generate_test_data
+        CSV.open(path.join(CSV_FILENAME), "wb") do |csv|
+          csv << ["message_body"]
+
+          100.times do
```

Comment:
We will need a lot more than 100 here because duplicate SMSes are rejected within a time window of 12 hours. So basically if we want to send 200k SMSes we really need 200k lines in this file! Can you perhaps make the number of sample responses an argument to both of these tasks that defaults to 100?